modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrderTypes.testcase

```diff
@@ -254,6 +383,338 @@ definition {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			FrontStore.addInitializerProductToCart(
+				orderType = "Test Order Type 1",
+				productName = "U-Joint",
+				productQuantity = 1);
+		}
+
+		task ("When the buyer completes the checkout with Order Type 1") {
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+
+			CommerceCheckout.initializersCheckout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = 1,
+				productUnitPrice = 24,
+				shippingMethod = "Standard");
+		}
+
+		task ("And the buyer adds a product to the cart again") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
+
+			Click(
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+		}
+
+		task ("Then the Select Order Type popup is shown again") {
+			AssertElementPresent(locator1 = "IFrame#MODAL_IFRAME");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11573. CanSelectOrderTypeWithEligibilityForANewOrder"
+	@priority = 3
+	test CanSelectOrderTypeWithEligibilityForANewOrder {
+		property portal.acceptance = "false";
+
+		task ("Given three active order types are created") {
+			for (var orderTypeNumber : list "1,2,3") {
+				CommerceJSONOrdersAPI._addOrderType(
+					active = "true",
+					orderTypeDescription = "Test Order Type Description ${orderTypeNumber}",
+					orderTypeName = "Test Order Type ${orderTypeNumber}");
+			}
+		}
+
+		task ("And the channel eligibility for two order types is set to Minium Portal, one is set to non-Minium Portal") {
+			for (var orderTypeNumber : list "1,2") {
+				CommerceOrders.openOrderTypesAdmin();
+
+				CommerceNavigator.gotoEntry(entryName = "Test Order Type ${orderTypeNumber}");
+
+				CommerceEntry.gotoMenuTab(menuTab = "Eligibility");
+
+				CommerceEntry.setEntryEligibility(channelsEligibility = "Minium Portal");
+			}
+
+			CommerceJSONChannelsAPI._addChannel(
+				channelName = "Test Channel",
+				channelType = "site",
+				currencyCode = "USD");
+
+			CommerceOrders.openOrderTypesAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Test Order Type 3");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Eligibility");
+
+			CommerceEntry.setEntryEligibility(channelsEligibility = "Test Channel");
+		}
+
+		task ("When the buyer adds a product to the cart") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			Click(
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+		}
+
+		task ("Then the Select Order Type popup is shown with the two order types which setten for Minium Portal channel only") {
```

Comment:
"Then the Select Order Type popup is shown with the two order types eligible for Minium Portal channel"